common/helpers/move/set-missing-to-location.js

```diff
@@ -0,0 +1,21 @@
+const i18n = require('../../../config/i18n')
+
+function setMissingToLocation(move) {
+  if (move.to_location) {
+    return move
+  }
+
+  const hasOwnLabel = ['prison_recall', 'video_remand']
+
+  return {
+    ...move,
+    to_location: {
+      key: `unknown__${move.move_type}`,
+      title: hasOwnLabel.includes(move.move_type)
+        ? i18n.t(`fields::move_type.items.${move.move_type}.label`)
+        : i18n.t('fields::move_type.items.unknown.label'),
+    },
+  }
+}
+
+module.exports = setMissingToLocation
```

Comment:
Note: This is only a draft, so tests to come

Reply:
Yep, the review was more for feedback on the approach so if anything needed majorly refactoring I didn't have to spend lots of time changing all the tests too :)